translator/config/schema.json

```diff
@@ -468,6 +468,11 @@
                 "minLength": 1,
                 "maxLength": 255
               }
+            },
+            "append_dimensions": {
+              "type": "object",
+              "description": "Adds Amazon EC2 metric dimensions to all metrics collected by the agent, we only support fixed key value pair now: ImageId:{aws:ImageId},InstanceId:{aws:InstanceId},InstanceType:{aws:InstanceType},AutoScalingGroupName:{aws:AutoScalingGroupName}. ",
```

Comment:
This isnt right. At the plugin level, we dont support these placeholders.
This should follow `#/definitions/generalAppendDimensionsDefinition`